spectastic/request.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, body, headers, query, path):
         if not hasattr(query, 'iteritems'):
             raise ValueError('Query is not dictionary-like')
 
-        if 'content-type' in headers:
+        if 'content-type' in headers and method != 'GET':
             value = headers['content-type']
             if value == 'application/json':
                 if isinstance(body, basestring) and body == '':
```

Comment:
I'm guessing this doesn't work because `body` is none for a GET request. Instead of updating the signature of BasicRequest and propagating the request method can we just check for that?

Reply:
Sounds good to me.